app/components/Article/index.tsx

```diff
@@ -159,11 +207,17 @@ const ArticleFooter = (question: Question) => {
       <span>Did this page help you?</span>
 
       <CompositeButton>
-        <Button className="secondary" action={() => alert('Like')}>
+        <Button
+          className={['secondary', actionIds['helpful'] ? 'focused' : ''].join(' ')}
+          action={() => handleAction('helpful')}
+        >
           <ThumbUpIcon />
           <span className="teal-500">Yes</span>
         </Button>
-        <Button className="secondary" action={() => alert('Dislike')}>
+        <Button
```

Comment:
why not just do
```
import {Action, ActionType} from '~/routes/questions.actions'
(...)
<CompositeButton>
    <Action pageid={pageid} showText={false} actionType={ActionType.LIKE} />
    <Action pageid={pageid} showText={false} actionType={ActionType.DISLIKE} />
</CompositeButton>
```
with the appropriate styling/component updates?

Reply:
There were mentions of backwards compatibility.  "The appropriate styling" would make it unfit on the old domain.